Tars/Tars/ViewController/InfoViewController.swift

```diff
@@ -59,59 +61,65 @@ class InfoViewController: UIViewController {
             if planet.planetKoreanName == $0.planetName {
-                customPlanetInfoChapterOne.chapter.text = "Chapter 1"
-                customPlanetInfoChapterOne.planetInfoTitle.text = $0.planetTitle1
-                customPlanetInfoChapterOne.planetInfoContents.text = $0.planetContents1
-                print("$0.planetTitle1: \($0.planetTitle1)")
-                customPlanetInfoChapterTwo.chapter.text = "Chapter 2"
-                customPlanetInfoChapterTwo.planetInfoTitle.text = $0.planetTitle2
-                customPlanetInfoChapterTwo.planetInfoContents.text = $0.planetContents2
-                customPlanetInfoChapterThree.chapter.text = "Chapter 3"
-                customPlanetInfoChapterThree.planetInfoTitle.text = $0.planetTitle3
-                customPlanetInfoChapterThree.planetInfoContents.text = $0.planetContents3
+                customPlanetInfoChapterOne.setInfoContents(chapter: "Chapter 1", title: $0.planetTitle1, contents: $0.planetContents1)
+                customPlanetInfoChapterTwo.setInfoContents(chapter: "Chapter 2", title: $0.planetTitle2, contents: $0.planetContents2)
+                customPlanetInfoChapterThree.setInfoContents(chapter: "Chapter 3", title: $0.planetTitle3, contents: $0.planetContents3)
             }
         }
-//        for i in 0..<planetContentsList.count {
-//            if planet.planetKoreanName == planetContentsList[i].planetName {
-//                customPlanetInfoChapterOne.chapter.text = "Chapter 1"
-//                customPlanetInfoChapterOne.planetInfoTitle.text = planetContentsList[i].planetTitle1
-//                customPlanetInfoChapterOne.planetInfoContents.text = planetContentsList[i].planetContents1
-//                customPlanetInfoChapterTwo.chapter.text = "Chapter 2"
-//                customPlanetInfoChapterTwo.planetInfoTitle.text = planetContentsList[i].planetTitle2
-//                customPlanetInfoChapterTwo.planetInfoContents.text = planetContentsList[i].planetContents2
-//                customPlanetInfoChapterThree.chapter.text = "Chapter 3"
-//                customPlanetInfoChapterThree.planetInfoTitle.text = planetContentsList[i].planetTitle3
-//                customPlanetInfoChapterThree.planetInfoContents.text = planetContentsList[i].planetContents3
-//            }
-//        }
-        stackView.distribution = .fillEqually
+
+        stackView.distribution = .fillProportionally
         stackView.axis = .vertical
+        stackView.alignment = .leading
         return stackView
     }()
     
+    // MARK: - Lifecycle
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
         view.backgroundColor = .black
-        [sceneView, customInfoStackView].forEach { view.addSubview($0) }
+        [sceneView, customInfoScrollView].forEach { view.addSubview($0) }
+        customInfoScrollView.addSubview(customInfoStackView)
         configureConstraints()
+        navigationItem.title = planet.planetKoreanName
```

Comment:
p4;
혹시 이렇게 네비게이션 타이틀을 planetKoreanName으로 하셨는데, 혹시 로컬라이징을 할 때 어떻게 하면 될까 의문을 가져서 이렇게 코멘트 남깁니다!

영어이름이 `planet.planetEnglishName`이 별도로 있는 것으로 아는데, if문으로 하기에는 아쉽고,
만약 저라면 다음과 같이 할 것 같은데...  
```suggestion
navigationItem.title = planet.PlanetStrings.천체이름.localizedKey
```
혹시 어떻게 생각하시는지 궁금합니다~ 🥹

Reply:
로컬라이징의 경우를 고려하지 못했네요...! 
localizedKey가 지금 어떤 형태로 되어있는지 확인 먼저 해봐야 할 것 같습니다. 
짚어주셔서 감사합니다 :)